plugins/wazo_fanvil/common/common.py

```diff
@@ -518,6 +519,10 @@ def _add_phonebook_url(self, raw_config: dict[str, Any]) -> None:
             hasattr(plugins, 'add_xivo_phonebook_url')
             and raw_config.get('config_version', 0) >= 1
         ):
+            phonebook_url = raw_config.get('XX_xivo_phonebook_url')
+            if not phonebook_url:
+                return
+
```

Comment:
If the goal of this is to check if the URL is present before setting the `.replace`, I think this code should be after `plugins.add_xivo_phonebook_url`. `XX_xivo_phonebook_url` is not supposed to be present before executing `plugins.add_xivo_phonebook_url`, so I think it will always return here. Am I wrong?

En Français : si le but de ce bout de code est pour ne rien faire car le `.replace` plus bas plante, je crois que ça devrait être fait après le `plugins.add_xivo_phonebook_url`. À priori il n'est pas censé y avoir de `XX_xivo_phonebook_url` avant que `plugins.add_xivo_phonebook_url` ne soit exécuté, donc ça va toujours return.
Me trompe-je? :smile:

Reply:
done